roles/cobbler_profile/defaults/main.yml

```diff
@@ -86,7 +86,7 @@ distros:
       kernel_options_post: "pci=realloc=off console=tty0 console=ttyS1,115200"
   "Ubuntu-16.04-server-x86_64":
       iso: "http://releases.ubuntu.com/16.04/ubuntu-16.04.6-server-amd64.iso"
-      sha256: a06cd926f5855d4f21fb4bc9978a35312f815fbda0d0ef7fdc846861f4fc4600
+      sha256: 16afb1375372c57471ea5e29803a89a5a6bd1f6aabea2e5e34ac1ab7eb9786ac 
```

Comment:
Nitpick: There's a trailing space after the sha.  Please fix, `git add .; git commit --amend --no-edit; git push -f origin wip-cobbler-838`